client/src/bundles/bundle.js

```diff
@@ -100,6 +100,7 @@ require('../legacy/AddToCampaignForm');
 require('../legacy/SecurityAdmin');
 require('../legacy/ModelAdmin');
 require('../legacy/ToastsContainer');
+require('../legacy/BootstrapComponent');
```

Comment:
We're registering tho logic to bootstrap the react components. This would only happen once in core.